build.sbt

```
@@ -28,7 +28,7 @@ ThisBuild / crossScalaVersions := {
 }
 ThisBuild / scalaVersion := crossScalaVersions.value.find(_.startsWith("2.13.")).get
 
-val inCI = Option(System.getenv("CI")).exists(_.contains("1"))
+val inCI = Option(System.getProperty("CI")).exists(_.contains("1"))
```

why not Scala's `sys.props` btw?

---

Why not `System.getProperty` lol? Same thing either way

---

Because, why scala? 😝 lol nbd, infrastructure is ugly anyway 🤣  

```suggestion
val inCI = sys.props.get("CI").exists(_.contains("1"))
```